src/multiparser.js

```diff
@@ -159,6 +163,62 @@ function fromHtmlParser2(path, options) {
   }
 }
 
+function wrapCssTemplate(quasisDoc, expressionDocs) {
+  const allReplaced = replacePlaceholders(quasisDoc, expressionDocs);
+  if (!allReplaced) {
+    throw new Error("Couldn't insert all the expressions");
```

Comment:
Could you use createError with the right location, this way if it fires people won't be as confused.